client/ayon_core/hosts/maya/api/lib.py

```diff
@@ -2641,12 +2641,24 @@ def reset_scene_resolution():
         None
     """
 
-    folder_attributes = get_current_project_folder()["attrib"]
+    # Set frame start/end
+    project_name = get_current_project_name()
+    folder_path = get_current_folder_path()
+    task_name = get_current_task_name()
 
+    folder_entity = ayon_api.get_folder_by_path(
+        project_name, folder_path, fields={"id"})
+    task_entity = ayon_api.get_task_by_name(
+        project_name,
+        folder_entity["id"],
+        task_name,
+        fields={"attrib"}
+    )
+    task_attributes = task_entity["attrib"]
```

Comment:
This should now be able to use `get_current_task_entity()` since this PR: https://github.com/ynput/ayon-core/pull/353

See e.g. the logic here: https://github.com/ynput/ayon-core/pull/358/files#diff-d3957520925f7a55982d51ee662c9f5432aaa46f67cdff06858b2d3a9575598eR2639